uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsenceTypeTest.php

```diff
@@ -4,7 +4,20 @@
 
 class CRM_HRLeaveAndAbsences_BAO_AbsenceTypeTest extends CiviUnitTestCase {
 
+  private $allColors = [
+      '#5A6779', '#E5807F', '#ECA67F', '#8EC68A', '#C096AA', '#9579A8', '#42B0CB',
+      '#3D4A5E', '#E56A6A', '#FA8F55', '#6DAD68', '#B37995', '#84619C', '#2997B3',
+      '#263345', '#CC4A49', '#D97038', '#4F944A', '#995978', '#5F3D76', '#147E99',
+      '#151D2C', '#B32E2E', '#BF561D', '#377A31', '#803D5E', '#47275C', '#056780'
+  ];
+
+  private $_tables = [
+    'civicrm_hrleaveandabsences_notification_receiver',
+    'civicrm_hrleaveandabsences_absence_type',
+  ];
+
   public function setUp() {
+    $this->clearTables();
```

Comment:
also i think it is better to use the already defined method to clear tables ... **quickCleanup()** ... 

  function tearDown() {
    $tablesToTruncate = array('table_1' , 'table_2' );
    $this->quickCleanup($tablesToTruncate);
}